cfn-resources/autogen/README.md

```diff
@@ -0,0 +1,40 @@
+# Auto Generate cfn template and generic GO Code
+
+## How to generate Resource setup and Respective Go template for a new Resource
+
+### Step 1:
+
+Update mapping.json in schema folder with new resource details
+
+eg:
+    `{
```

Comment:
Can you double check how this looks in preview? the formatting in off, i belive should be 1 block of code not several lines 

![image](https://user-images.githubusercontent.com/47370943/204931534-7d064e19-70b5-4cdd-b2fa-2e7b46adeaf2.png)